### DISCUSSION TITLE
Is there a “ Recommended configuration” for scrub?

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12333] [Q&A]
```
osd_scrub_begin_hour
osd_scrub_end_hour
osd_scrub_begin_week_day
osd_scrub_end_week_day
```

if we don't want ceph cluster run "scrub" and "deep scrub" at work time, how to avoid it, and is there is recommended configuration for it?

**Comment by travisn** [comment]
Rook doesn't set those, it's just a ceph default behavior. If you want to change it, you can set those with `ceph config set` in the toolbox, or with the ceph config overrides.

**Reply by zhucan** [reply]
I had set it,  but it seems invalid, under work time, it still "scrub" and "deep scrub"
